scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml

```diff
@@ -48,3 +48,46 @@ HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm_ThermalVias:
     # grid: [1.5, 1.5]
     # bottom_pad_size:
     paste_avoid_via: False
+
+HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.3x2.3mm_ThermalVias:
```

Comment:
This seems completely unrelated.
Please remove.

Reply:
My mistake... I removed it.